hi-ml-histopathology/src/histopathology/datasets/panda_tiles_dataset.py

```diff
@@ -41,14 +41,20 @@ def __init__(self,
         super().__init__(root=Path(root) / self._RELATIVE_ROOT_FOLDER,
                          dataset_csv=dataset_csv,
                          dataset_df=dataset_df,
-                         train=None)
+                         train=None,
+                         validate_columns=False)
         if occupancy_threshold is not None:
             self.dataset_df: pd.DataFrame
             dataset_df_filtered = self.dataset_df.loc[
                 self.dataset_df['occupancy'] > occupancy_threshold
             ]
             self.dataset_df = dataset_df_filtered
 
+        # Change "left" --> "tile_x" and "top" --> "tile_y"
+        # to be consistent with TilesDataset `TILE_X_COLUMN` and `TILE_Y_COLUMN`
+        self.dataset_df.rename(columns={"left": "tile_x", "top": "tile_y"}, inplace=True)
```

Comment:
Thanks both, I have now made the suggested change. @dccastro since I am directly renaming the dataframe column, I haven't explicitly defined `PandaTilesDataset.TILE_X_COLUMN`, we are using `TilesDataset.TILE_X_COLUMN` in other places.

Reply:
@harshita-s, please ignore my comment above, you're right about those being undefined.